src/mapbox/extended_nav_control.js

```diff
@@ -10,11 +10,19 @@ export default class ExtendedControl {
     this.bottomButtonGroup = document.createElement('div');
 
     const buttonClass = 'icon-plus map_control_group__button__zoom';
-    this._zoomInButton = this._createButton(buttonClass, 'Zoom In', () => this._map.zoomIn());
+
+    this._zoomInButton = this._createButton(
+      buttonClass,
+      'Zoom In',
+      () => this._map.zoomIn(),
+      'Zoom +'
```

Comment:
There is no added value here to use a different text in `title` and `aria-label`, you can just re-use the same string for both.

Reply:
Ok, title is set according to ariaLabel now